Program/23BCA281.py

```diff
@@ -0,0 +1,56 @@
+###[ 7 ] Create table contact(fname,laname,contact,email,city). Perform following operation on it
+##1. Insert Contact 8 records
+##2. Delete Contact¶"""
+
+
+import sqlite3 as sq
+
+#Connect to the py.db databse
+conn=sq.connect("D:\\sqlite\\csv\\py.db")
+cur=conn.cursor()
+#Create table contact
+cur.execute('''
+CREATE TABLE IF NOT EXISTS contact (
+    fname TEXT,
+    lname TEXT,
+    contact TEXT,
+    email TEXT,
+    city TEXT
+)
+''')
+#Insert 8 records in the table
+records = [

```

Comment:
Why did you list it while inserting the record?

Reply:
> Why did you list it while inserting the record?

1. Here I use list "[]". 
2. It's easier to maintain and see all records in one place.